examples/handlers/set_alarm.js

```diff
@@ -1,15 +1,17 @@
 
 module.exports = function (session, next, data) {
 
-  var intent = session.dialogData[data.source];
+  var intent = session.privateConversationData[data.source];
   var alarmTime = null;
   if (intent.actions[0].parameters[0].name == "time") {
-
-    alarmTime = intent.actions[0].parameters[0].value[0].resolution.time;
+    // alarmTime = intent.entities...
+    // use intent.entities to extract relevant information                
+    // assuming extracted alarmTime                                       
+    alarmTime = '2016-10-10 10:10';         
```

Comment:
You can add the old code as a comment to show a sample of how to get the resolved time